src/test/java/org/folio/search/service/setter/item/ItemTypedCallNumberProcessorTest.java

```diff
@@ -66,8 +66,6 @@ void getFieldValue_emptyCallNumberAfterProcessing(String effectiveShelvingOrder,
 
   @Test
   void getFieldValue_emptyCallNumberIfNotSystemOrNotLocal() {
-    when(referenceDataService.fetchReferenceData(CALL_NUMBER_TYPES, SOURCE, LOCAL_CALL_NUMBER_TYPES_SOURCES))
-      .thenReturn(emptySet());
```

Comment:
Why the mock is not needed anymore?

Reply:
I'm not sure was it was needed at all, Mockito returns empty collection for collections by default.
Also test was failing with this stubbing for me (saying that it's unnecessary).

Reply:
Strange. Okay